src/components/UserSettingsProvider.js

```diff
@@ -15,7 +15,10 @@ const UserSettingsProvider = ({ children }) => {
                 userSettings: userSettingsQuery,
             })
 
-            setSettings(userSettings)
+            setSettings({
+                ...userSettings,
+                displayProperty: userSettings.keyAnalysisDisplayProperty,
```

Comment:
If there is time, I would prefer to not duplicate this property in the result (now it has both `displayProperty` and `keyAnalysisDisplayProperty`).  You can see it being used in FilterDialog.  One approach could be to give better names for all the returned settings (similar is done in the system settings (api/systemSettings and SystemSettingsProvider). If there isn't time for this, we can do it later